mkosi.uki-profiles/profile1.conf

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[UKIProfile]
+Profile=
+        ID=profile1
```

Comment:
Looks like mkosi images fail to build on Fedora 40 with
```sh
All done.
‣  Generating UKI profile 'profile1'
ukify: error: unrecognized arguments: --profile @/home/vagrant/.cache/mkosi/mkosi-workspace-3jg1kp58/uki-profiles/profile1.profile
‣ "python3 /usr/bin/ukify build --profile @/home/vagrant/.cache/mkosi/mkosi-workspace-3jg1kp58/uki-profiles/profile1.profile --efi-arch x64 --stub /work/home/vagrant/.cache/mkosi/mkosi-workspace-3jg1kp58/root/usr/lib/systemd/boot/efi/addonx64.efi.stub --output /work/home/vagrant/.cache/mkosi/mkosi-workspace-3jg1kp58/uki-profiles/profile1.efi --cmdline @/work/home/vagrant/.cache/mkosi/mkosi-workspace-3jg1kp58/cmdline" returned non-zero exit code 2.
```

Reply:
@evverx You have to build systemd on the host and configure mkosi to use tools from the build directory, see HACKING.md for a complete example.

Reply:
@DaanDeMeyer it still says
> mkosi will build an image for the host distro by default. First, run mkosi genkey to generate a key and certificate to be used for secure boot and verity signing. After that is done, it is sufficient to type mkosi in the systemd project directory to generate a disk image you can boot either in systemd-nspawn or in a UEFI-capable VM

and it worked on Fedora 40 up until I pulled this PR. If it can't be used without pointing mkosi to `build` it should probably be mentioned.

Reply:
Given that the mkosi part rolls forward along with systemd maybe it should use that stuff by default instead of whatever is installed on the host.

Reply:
I saw https://github.com/systemd/systemd/pull/34854 and I think it probably makes it easier to figure out what to do. I ended up deleting the `mkosi.uki-profiles` directory though because I don't need those things. I think it would be great to make it work like that by default but it can probably break something else.